nebu/cli/get.py

```diff
@@ -26,6 +29,12 @@ def get(ctx, env, col_id, col_version, output_dir, book_tree):
 
     base_url = build_archive_url(ctx, env)
 
+    version = None
+    if col_version.count('.') > 1:
```

Comment:
Interesting; is it practically the same as saying `if col_version != 'latest'`?

Reply:
nope, since the usual case is `col12345 1.14`, which ends up retrieving `1.14.<largest-minor>`. This is to enable getting `1.14.10` when `1.14.11` exists.